docs/protocol/agents/validators.mdx

```diff
@@ -0,0 +1,27 @@
+# Validators
+
+Validators are responsible for observing the [Mailbox](../mailbox.mdx) contract and, if required, signing its merkle root to facilitating message transmission to remote chains.
+
+Unlike many other protocols, Hyperlane does **not** have an enshrined validator set. Anyone is free to run a validator and help contribute to securing interchain messages.
+
+:::info
+Want to run a validator? Follow the [validators guide](../../operate/validators/run-validators.mdx).
+:::
+
+For messages using a [Multisig ISM](../../reference/ISM/multisig-ISM-interface.mdx), validators read the current merkle root by calling `MerkleTreeHook.latestCheckpoint()`.  Once a root has achieved sufficient [finality](https://medium.com/mechanism-labs/finality-in-blockchain-consensus-d1f83c120a9a), validators are expected to sign it and post their signature to highly available storage so that it can be aggregated by [relayers](./relayers.mdx).
+
+```solidity
+/**
+  * @notice Returns the latest checkpoint for validators to sign.
+  * @return root Latest checkpointed root
+  * @return index Latest checkpointed index
+  */
+function latestCheckpoint()
+  external
+  view
+  returns (bytes32 root, uint256 index);
+```
+
+Validators that sign anything other than a finalized checkpoint risk compromising the safety of the protocol. If they are participating in proof of stake, this may result in their stake being slashed.
+
+Hyperlane is developing the validator as a Rust binary that can be easily run by anyone. Operationally, validators need to run this binary and a node for the chain that they are validating for. We hope that the majority of Hyperlane validators will come from each chain's existing node operator community.
```

Comment:
Was this paragraph in the docs already? Validators don't _need_ to run a chain node, since they can just use an rpc provider to send their validator announce tx and read from the mailbox / merkle tree hook

Reply:
yes it was, might have been some conflation from polygon specific comments relating to the archive node

Reply:
```suggestion
Hyperlane is developing the validator as a binary implemented in Rust that can be easily run by anyone. Operationally, validators need to run this binary and have access to an RPC provider or node for the chain that they are validating for. We hope that the majority of Hyperlane validators will come from each chain's existing node operator community.
```
updating to